libs/npm/core/src/utils/elastic/generateTextQuery.ts

```diff
@@ -10,17 +10,18 @@ export const generateTextQuery = (
     // give an arbitrary weight to the headline, so if it's found there
     // it gets a slightly higher score, as opposed to other fields
     return generateSimpleQueryString(
-      ['headline^5', 'byline', 'summary', 'body'],
+      ['headline^5', 'byline', 'summary', 'body', 'series.name'],
       settings.search,
       settings.defaultSearchOperator,
     );
   }
 
   let fields: string[] = [];
-  if (settings.inByline || settings.inStory || settings.inHeadline) {
+  if (settings.inByline || settings.inStory || settings.inHeadline || settings.inProgram) {
     if (!!settings.inByline) fields = [...fields, 'byline'];
     if (!!settings.inStory) fields = [...fields, 'summary', 'body'];
     if (!!settings.inHeadline) fields = [...fields, 'headline'];
+    if (!!settings.inProgram) fields = [...fields, 'series.name'];
```

Comment:
Does this actually work?  Kind amazing it's so simple to query another field.

Reply:
![image](https://github.com/bcgov/tno/assets/26801490/a6b9c2b3-7684-4d7e-879f-85a2d10373c9)
Totally. Never did that, and it was so easy to implement.